app/models/user.rb

```diff
@@ -0,0 +1,22 @@
+class User < ApplicationRecord
+    has_many :events, foreign_key: "creator_id"
+
+    has_many :attendances
+    has_many :attended_events, through: :attendances, source: "event"
+
+    has_many :created_invites, class_name: "Invitation", foreign_key: "invitor_id"
```

Comment:
To have a more meaningful association, I suggest we do away with this particular one altogether. I see that you are using this in your invitations_controller. The better way to create an invitation is anchor it on the `invitation` itself and not on the user. See my comment on the `invitations_controller.rb` file